src/Chainweb/Pact/Backend/Utils.hs

```diff
@@ -212,14 +212,14 @@ chainwebPragmas :: [Pragma]
 chainwebPragmas =
   [ "synchronous = NORMAL"
   , "journal_mode = WAL"
-  , "locking_mode = EXCLUSIVE"
+  -- , "locking_mode = EXCLUSIVE"
+  , "locking_mode = NORMAL"
```

Comment:
Add a comment about the effects of the setting and the reason why it was changed